app/scripts/filters/resources.js

```diff
@@ -1093,6 +1105,13 @@ angular.module('openshiftConsole')
       return _.find(_.get(apiObject, 'status.conditions'), {type: type});
     };
   })
+  // returns true/false based on the presence of a Ready status conditions
+  // statusConditionReady(obj)
+  .filter('statusConditionReady', function(statusConditionFilter) {
```

Comment:
You could also have two names for the filter. Call this `isServiceInstanceReady` and then have an `isBindingReady` filter that returns `isServiceInstanceReadyFilter`.

These might be useful in origin-web-common, but don't worry about that for this PR.